### PR TITLE
Add piwik to criteria page

### PR DESCRIPTION
**Pull request by ch-mcabrera**
Code changes to add piwik to the criteria page when the radio buttons are clicked.
This change has followed similar changes made in the transaction web. See Pr for further info:  https://github.com/companieshouse/transactions.web.ch.gov.uk/pull/16)

Resolves: SFA-653

### CODE REVIEW

File: src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/BalanceSheetControllerTests.java [hunk 1]
```diff
@@ -68,6 +68,8 @@
 
     private static final String BACK_BUTTON_MODEL_ATTR = "backButton";
 
+    private static final String TEMPLATE_NAME_MODEL_ATTR = "templateName";
```
Comment (BSaunders95) [inline src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/BalanceSheetControllerTests.java]: Can you assert this attribute exists when the page is redisplayed after an error?

File: src/main/resources/templates/fragments/piwik.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>Piwik</title>
+</head>
+<body>
+<div th:fragment="piwik">
+  <script th:inline="javascript">
+                (function() {
+                    bindPiwikListener('web', /*[[${@environment.getProperty('piwik.url')}]]*/, /*[[${@environment.getProperty('piwik.siteId')}]]*/);
+                })();
```
Comment (marcransome) [inline src/main/resources/templates/fragments/piwik.html]: Is there a reason the function call arguments are commented out here? This isn't valid JavaScript: `Uncaught SyntaxError: Unexpected token `.

File: src/main/resources/templates/fragments/piwik.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>Piwik</title>
+</head>
+<body>
+<div th:fragment="piwik">
+  <script th:inline="javascript">
+                (function() {
+                    bindPiwikListener('web', /*[[${@environment.getProperty('piwik.url')}]]*/, /*[[${@environment.getProperty('piwik.siteId')}]]*/);
```
Comment (BSaunders95) [inline src/main/resources/templates/fragments/piwik.html]: Can the first argument here be changed to 'company-accounts' for the clarity in the Piwik dashboard?

**Comment by BSaunders95** [comment]
As part of these changes, a separate PR must be raised for the `enable-piwik.js` script to remove ids from the url paths. In the `setCustomUrls` method, we should replace company number with `_number`

File: src/main/resources/templates/fragments/piwik.html (hunk 3 above)
Comment (ch-mcabrera) [reply src/main/resources/templates/fragments/piwik.html]: Perfect, I will do. I wasn't sure what the argument i had to used

**Pull request merged**